graphql/src/lib.rs

```diff
@@ -18,18 +18,18 @@ mod handler;
 
 use actix_web::{
```

Comment:
Where does the term `Extractor` come from?

Reply:
From actix guide. They call `impl FromRequest` as extractor: https://actix.rs/docs/extractors/

Reply:
Nice